test/runtime/kafka.go

```diff
@@ -139,4 +139,38 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 		res = vm.Exec(consumer(disallowTopic, MaxMessages))
 		res.ExpectFail("Kafka consumer can access to disallowTopic")
 	})
+
+	It("Kafka Policy Role Ingress", func() {
+		_, err := vm.PolicyImportAndWait(vm.GetFullPath("Policies-kafka-Role.json"), 300)
+		Expect(err).Should(BeNil())
+
+		endPoints, err := vm.PolicyEndpointsSummary()
+		Expect(err).Should(BeNil())
+		Expect(endPoints[helpers.Enabled]).To(Equal(1))
+		Expect(endPoints[helpers.Disabled]).To(Equal(2))
+
+		createTopic(allowedTopic)
+		createTopic(disallowTopic)
+
+		res := vm.ContainerExec("client",
+			"/opt/kafka/bin/kafka-topics.sh --list --zookeeper zook:2181")
+		res.ExpectSuccess("Cannot get kafka topics")
+
+		By("Allowed topic")
```

Comment:
Write "By" as it was a sentence

"By allowing topic foo" -> `By("Allowing topic foo")`

Reply:
done